mousestyles/tests/test_path.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import pytest
+
+from mousestyles import data
+from mousestyles.path_index import path_index
+
+
+def test_path_input():
+    movement = data.load_movement(0, 0, 0)
+    # checking functions raise the correct errors
+    with pytest.raises(ValueError) as excinfo:
+        path_index(movement, -1, -1)
+    assert excinfo.value.args[0] == "Input values need to be positive"
+    with pytest.raises(TypeError) as excinfo:
+        path_index(movement, 1.5, 1.5)
+    assert excinfo.value.args[0] == "Input values need to be integer"

```

Comment:
This could be way off, but do tests need any sort of docstrings? Or just the expected output?